test/path.jl

```diff
@@ -171,6 +171,9 @@
         @test string(splitdrive(S(homedir()))...) == homedir()
         @test splitdrive("a\nb") == ("", "a\nb")
 
+        @test splitdir("a/\xfe/\n/b/c.ext") == ("a/\xfe/\n/b", "c.ext")
+        @test splitext("a/\xfe/\n/b/c.ext") == ("a/\xfe/\n/b/c", ".ext")
```

Comment:
Thought someone might be interested to note that this had previously been encountered as a bug and fixed only for splitdrive and only for windows by https://github.com/JuliaLang/julia/pull/32543